pkg/services/oauthtoken/oauth_token.go

```diff
@@ -35,6 +35,7 @@ var (
 	ErrNoRefreshTokenFound = errors.New("no refresh token found")
 	ErrNotAnOAuthProvider  = errors.New("not an oauth provider")
 	ErrCouldntRefreshToken = errors.New("could not refresh token")
```

Comment:
nit: Looks like `ErrCouldntRefreshToken` is not used anywhere anymore.

Reply:
Thanks, I'm removing `ErrCouldntRefreshToken` as part of https://github.com/grafana/grafana/pull/98171.